### PR TITLE
Add about.md

### PR DESCRIPTION
**Pull request by yesroh**
Added the file about.md

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+H1
+# MWhat is PHP?
```
Comment (kytrinyx) [inline docs/ABOUT.md]: We don't need the title, since the website takes care of that for us.

File: docs/ABOUT.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+H1
```
Comment (kytrinyx) [inline docs/ABOUT.md]: I think this is a stray bit of text. Would you remove it, please?

File: docs/ABOUT.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+What distinguishes PHP from something like client-side JavaScript is that the code is executed on the server, generating HTML which is then sent to the client. The client would receive the results of running that script, but would not know what the underlying code was. You can even configure your web server to process all your HTML files with PHP, and then there's really no way that users can tell what you have up your sleeve.
+
+The best things in using PHP are that it is extremely simple for a newcomer, but offers many advanced features for a professional programmer. Don't be afraid reading the long list of PHP's features. You can jump in, in a short time, and start writing simple scripts in a few hours.
+
+
+Although PHP's development is focused on server-side scripting, you can do much more with it. Read on, and see more in the What can PHP do? section, or go right to the introductory tutorial if you are only interested in web programming.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Did you copy/paste this text from somewhere? Is that text copyrighted?

**Comment by kytrinyx** [comment]
I like the tone and contents of this. I think we can delete the code example, but we can link to a more comprehensive introduction for people who are interested.

File: docs/ABOUT.md (hunk 3 above)
Comment (kytrinyx) [inline docs/ABOUT.md]: OK, with a bit of searching, I found it. http://php.net/manual/en/intro-whatis.php

**Comment by kytrinyx** [comment]
We can't copy/paste from the PHP manual, but we could potentially quote parts of it with proper sitation and a link back.

Would you mind reworking this in your own words?

**Comment by yesroh** [comment]
OK no problems. I did not know the rules. I will make a new presentation

**Comment by petemcfarlane** [comment]
closing, updated in #99 (closed) and #100

**Pull request closed without merging**